functions/helloworld/HelloWorld.Tests/FunctionUnitTest.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_http_unit_test]
```

Comment:
Ohh, cool, we are adding docs on testing?

Reply:
Yup! I may need to revisit these tests based on a new understanding of what the samples team considers unit vs integration, but let's get them in first :)